library/pod-security-policy/users/template.yaml

```diff
@@ -15,60 +18,100 @@ spec:
           properties:
             runAsUser:
               type: object
+              description: "Controls which user ID values are allowed in a Pod or container level SecurityContext."
               properties:
                 rule:
                   type: string
+                  description: "A strategy for applying the runAsUser restriction."
+                  enum:
+                    - MustRunAs
+                    - MustRunAsNonRoot
+                    - RunAsAny
                 ranges:
                   type: array
+                  description: "A list of user ID ranges affected by the rule."
                   items:
                     type: object
+                    description: "The range of user IDs affected by the rule."
                     properties:
                       min:
                         type: integer
+                        description: "The minimum user ID in the range, inclusive."
                       max:
                         type: integer
+                        description: "The maximum user ID in the range, inclusive."
             runAsGroup:
               type: object
+              description: "Controls which group ID values are allowed in a Pod or container level SecurityContext."
```

Comment:
I don't think Pod is capitalized, and container-level should be hyphenated when used as an adjective. "Controls which group ID values are allowed in a pod or container-level SecurityContext."

Reply:
Pod capitalized refers to the Kubernetes resource, so it is correct for it to be capitalized.

Reply:
Looks like [k8s docs](https://kubernetes.io/docs/concepts/workloads/pods/) use it like a proper noun.